.github/workflows/ci.yml

```
@@ -61,10 +61,10 @@ jobs:
           python -m spacy download fr_core_news_sm
       - name: Install dependencies (latest versions)
         if: ${{ matrix.deps_versions == 'deps-latest' }}
-        run: pip install --upgrade pyarrow huggingface-hub dill
+        run: pip install --upgrade pyarrow dill
```

to apply once hfh 0.18 is released

```suggestion
        run: pip install --upgrade pyarrow huggingface-hub dill
```

---

it's released :)